agent/src/main/java/com/teamscale/jacoco/agent/options/AgentOptions.java

```diff
@@ -426,6 +427,12 @@ public Path getOutputDirectory() {
 		return outputDirectory;
 	}
 
+	/** Creates a new temp file with the given prefix, extension and current timestamp and ensures that the parent folder actually exists. */
```

Comment:
It sounds like this would be better placed in FileSystemUtils or something similar. What do you think?

Reply:
In general I agree, but there is the outputDirectory and the way the file name is built up that I would both consider specific to the agent and therefore not suitable for a general purpose method in utils.

And the ensureDirectoryExists already is implemented in the general purpose part.

Would you instead prefer having a `FileSystemUtils.createTempFileWithTimestamp(Path parentDir, String prefix, String extension)`?

But I found even two more occurrences where replacing with `createTempFile` is possible.

Reply:
Ok, it's fine with me then.